client/scss/pages/_paperwork.scss

```diff
@@ -7,9 +7,13 @@
     border-bottom: 1px solid black;
   }
 
-  menu[type=toolbar] div.primary {
-    align-items: center;
+  menu[type=toolbar] {
```

Comment:
I agree that this is quite a bit of CSS, but it's a tough spot we're in since updating old styles might have side effects. I think we should gradually integrate into the other styles as we convert the rest of the site to this new look.